From b73c6b09c0c8d3d9162848a88fbe003d477ad0e2 Mon Sep 17 00:00:00 2001
From: Nathan Owens <ndowens@artixlinux.org>
Date: Wed, 30 Dec 2020 16:57:45 -0600
Subject: [PATCH] fix cross build

AC_TRY_RUN fails with cross-building
use AC_COMPILE_IFLESE instead
---
diff --git configure.in configure.in
index 4e75dd2..27c053c 100644
--- configure.in
+++ configure.in
@@ -83,7 +83,7 @@ void * thread_func(void * param) { return NULL; }
   , thread_ldflags="-pthread")
 
 AC_SUBST(large_file)
-AC_TRY_RUN([#ifndef _LARGEFILE64_SOURCE
+AC_COMPILE_IFELSE([#ifndef _LARGEFILE64_SOURCE
 #define _LARGEFILE64_SOURCE
 #endif
 #include <stdio.h>
-- 
2.30.0

